SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -3,7 +3,7 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
 | OpenMamba | [Openmamba GPLv2 License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt) | bash-completion |
 | TreasureData | [ASL 2.0 License](http://www.apache.org/licenses/LICENSE-2.0) | td-agent |
```

Comment:
Should the jitterentropy-library license also be referenced here? It's not currently being referenced for the OpenSSL patch.